httlemon/collections.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+from tinydb import TinyDB
```

Comment:
I'm not really familiar with `tinydb` but what's the issue behing using something like `sqlite` for instance?